packages/components/src/angle-picker-control/angle-circle.js

```diff
@@ -28,12 +28,11 @@ function AngleCircle( { value, onChange, ...props } ) {
 
 	const changeAngleToPosition = ( event ) => {
```

Comment:
I just noticed this is the handler for `onDragMove` and `onDragEnd` (in addition to being called by `onDragStart`) so it's executing `event.target.focus()` way more than necessary. Perhaps even the old `blur()` would work if it were only called from `onDragStart`. Not that I'm suggesting that `blur()` should be used.

Reply:
Good point! I didn't notice any performance penalty in my testing, but I suppose we could add a check before calling `.focus()` to only do that if the element isn't already focused?

Reply:
Definitely agree here. I noticed this as well that these events are tied together in a bit weird fashion. I'll land this as is to fix the issue for now, and we need to check how to separate/handle the drag events better.